examples/fluids/src/misc.c

```diff
@@ -465,3 +466,136 @@ PetscErrorCode IntArrayP2C(PetscInt num_entries, PetscInt **array_petsc, CeedInt
+      part_owned_dofs[1]             = gather_buffer[comm_size - 1];  // max
+      part_owned_dofs[2]             = gather_buffer[median_index];   // median
+      PetscReal part_owned_dof_ratio = (PetscReal)part_owned_dofs[1] / (PetscReal)part_owned_dofs[2];
+      PetscCall(PetscPrintf(comm, "    Owned free nodes                   : %" PetscInt_FMT ", %" PetscInt_FMT ", %" PetscInt_FMT ", %f\n",
+                            part_owned_dofs[0] / num_comp_q, part_owned_dofs[1] / num_comp_q, part_owned_dofs[2] / num_comp_q, part_owned_dof_ratio));
+    }
+
+    PetscCallMPI(MPI_Gather(&local_dofs, 1, PetscInt_MPI, gather_buffer, 1, PetscInt_MPI, 0, comm));
+    if (!rank) {
+      PetscCall(PetscSortInt(comm_size, gather_buffer));
+      part_local_dofs[0]             = gather_buffer[0];              // min
+      part_local_dofs[1]             = gather_buffer[comm_size - 1];  // max
+      part_local_dofs[2]             = gather_buffer[median_index];   // median
+      PetscReal part_local_dof_ratio = (PetscReal)part_local_dofs[1] / (PetscReal)part_local_dofs[2];
+      PetscCall(PetscPrintf(comm, "    Local nodes                        : %" PetscInt_FMT ", %" PetscInt_FMT ", %" PetscInt_FMT ", %f\n",
+                            part_local_dofs[0] / num_comp_q, part_local_dofs[1] / num_comp_q, part_local_dofs[2] / num_comp_q, part_local_dof_ratio));
+    }
+
+    PetscInt num_remote_roots_total = 0;
+    {
+      PetscSF            sf;
+      PetscInt           nrranks;
+      const PetscInt    *roffset, *rmine, *rremote;
+      const PetscMPIInt *rranks;
+      PetscCall(DMGetSectionSF(user->dm, &sf));
+      PetscCall(PetscSFGetRootRanks(sf, &nrranks, &rranks, &roffset, &rmine, &rremote));
+      for (PetscInt i = 0; i < nrranks; i++) {
+        if (rranks[i] == rank) continue;  // Ignore same-part global->local transfers
+        num_remote_roots_total += roffset[i + 1] - roffset[i];
+      }
+    }
+    PetscCallMPI(MPI_Gather(&num_remote_roots_total, 1, PetscInt_MPI, gather_buffer, 1, PetscInt_MPI, 0, comm));
+    if (!rank) {
+      PetscCall(PetscSortInt(comm_size, gather_buffer));
+      part_shared_dofs[0]             = gather_buffer[0];              // min
+      part_shared_dofs[1]             = gather_buffer[comm_size - 1];  // max
+      part_shared_dofs[2]             = gather_buffer[median_index];   // median
+      PetscReal part_shared_dof_ratio = (PetscReal)part_shared_dofs[1] / (PetscReal)part_shared_dofs[2];
+      PetscCall(PetscPrintf(comm, "    Shared nodes                       : %" PetscInt_FMT ", %" PetscInt_FMT ", %" PetscInt_FMT ", %f\n",
+                            part_shared_dofs[0] / num_comp_q, part_shared_dofs[1] / num_comp_q, part_shared_dofs[2] / num_comp_q,
+                            part_shared_dof_ratio));
+    }
+
+    if (!rank) PetscCall(PetscFree(gather_buffer));
+  }
+
+  PetscCall(PetscPrintf(comm, "(nodes == DoFs / %" PetscInt_FMT ")\n", num_comp_q));
```

Comment:
This is just dangling here in the otherwise pseudo-yaml table.

Reply:
Yeah, I intentionally put that there. Rather than having to describe our ad-hoc definition of "nodes" multiple times. Any suggestions on a better place to put it?

Reply:
I think it messes up the parse. We could call all the things `5-DoF nodes`, which is concise and still easy enough to parse when we have turbulence variables.